assets/js/blocks/mini-cart/mini-cart-contents/edit.tsx

```diff
@@ -124,26 +126,44 @@ const Edit = ( {
 
 	return (
 		<>
-			<InspectorControls key="inspector">
-				<PanelBody
-					title={ __( 'Dimensions', 'woo-gutenberg-products-block' ) }
-					initialOpen
-				>
-					<UnitControl
-						onChange={ ( value ) => {
-							setAttributes( { width: value } );
-						} }
-						value={ width }
-						units={ [
-							{
-								value: 'px',
-								label: 'px',
-								default: defaultAttributes.width.default,
-							},
-						] }
-					/>
-				</PanelBody>
-			</InspectorControls>
+			{ isFeaturePluginBuild() && (
+				<InspectorControls key="inspector">
+					<PanelBody
+						title={ __(
+							'Dimensions',
+							'woo-gutenberg-products-block'
+						) }
+						initialOpen
+					>
+						<UnitControl
+							onChange={ ( value ) => {
+								setAttributes( { width: value } );
+							} }
+							onBlur={ ( e: FocusEvent< HTMLInputElement > ) => {
+								if ( e.target.value === '' ) {
+									setAttributes( {
+										width: defaultAttributes.width.default,
+									} );
+								} else if (
+									Number( e.target.value ) < MIN_WIDTH
+								) {
+									setAttributes( {
+										width: MIN_WIDTH,
```

Comment:
This case is missing a `px` unit and the Mini Cart is not displayed at all in this case (has a width fallback to 0, because the property is incorrect).

```suggestion
										width: MIN_WIDTH + 'px',
```

Reply:
Good catch! I have incorporated your suggestion. Thanks!